site/_posts/2023-07-11-get-more-value-out-of-fusionauth.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: blog-post
+title: Get More Value Out of FusionAuth Using the APIs for User Maintenance
```

Comment:
We'll have Brad do an SEO review, but I think the good part of this tutorial is that you are using a windows form app, so why not mention it in the title?

Reply:
I think the windows form thing is good, but all of this applies to any form of customization.  I dow have windows form in the tags, but will defer to Brad.